### PR TITLE
Update our "master" because it's really old

### PR DESCRIPTION
**Pull request by torbengb**
As I'm reviewing #50 and submitting a message to YNAB (the company), I realize that we should link to the master ZIP file but that one is 4+ months old and has only a few formats. 

This request is meant to update the master. Should we give it some kind of version number? Anything else we should take into account?

### CODE REVIEW

**Comment by nocalla** [comment]
That's some changelog... I dunno are we ready to call it 1.0. But maybe we are, we have a working plugin system, support for several banks, and an automated testing system for newer code. 

I think we need to improve our tests so they verify that YNAB can import what's exported.

**Comment by torbengb** [comment]
> are we ready to call it 1.0
Why not, and was that even a question? :-) 
More tests are always good, but I don't see it as blocking a release.

**Comment by nocalla** [comment]
I'm okay with making this 1.0. We can make OFX output and better testing part of 1.1 maybe. The core functionality works.

File: bank2ynab.conf [hunk 1]
```diff
@@ -30,38 +33,165 @@ Plugin =
...
+Footer Rows = 0
+Input Columns = Date,Payee,skip,skip,skip,CDFlag,Amount,skip,Memo
+Date Format = YYYYMMDD
+Inflow or Outflow Indicator = 5,Af,Bij
+
+[NL Rabobank]
+# survey response #14
+Source Filename Pattern = transactions
+Source Filename Extension = .txt
+Source Has Column Headers = False
```
Comment (nocalla) [inline bank2ynab.conf]: Old format, we need to fix this

File: bank2ynab.conf [hunk 2]
```diff
@@ -30,38 +33,165 @@ Plugin =
...
+[SE Handelsbanken]
+Source Filename Pattern = kontotransactionlist
+Source Filename Extension = .xls
+Input Columns = skip,Date,Payee,Inflow,skip
+Source CSV Delimiter = ;
+Header Rows = 7
+Plugin = handelsbanken
+# don't delete source, so we can debug test
+Delete Source File = False
+
```
Comment (nocalla) [inline bank2ynab.conf]: Remove non-deletion

File: plugins/swedebank.py [hunk 3]
```diff
@@ -12,15 +12,15 @@ def read_data(self, file_path):
         delim = self.config["input_delimiter"]
```
Comment (nocalla) [inline plugins/swedebank.py]: Can we delete this plugin altogether?

Comment (torbengb) [reply plugins/swedebank.py]: Yes; it's not referenced in the config.

File: bank2ynab.conf (hunk 2 above)
Comment (torbengb) [reply bank2ynab.conf]: You're saying these lines should be removed, right? I'll do it now.

**Comment by torbengb** [comment]
@nocalla re-review if you please?

**Comment by nocalla** [comment]
We need to remove the test that specifically checks for the Swedbank plugin to make Travis happy!

**Comment by torbengb** [comment]
> remove the Swedbank test

Is it just a matter of removing "bank2ynab/test/test_swedebankPlugin.py" too, or is there more to it than that? I don't really understand how it's made.

**Comment by nocalla** [comment]
There's more to it then that I think. I'll tackle it soon, just got a lot
going on this week and next week.

**Comment by torbengb** [comment]
My rule #1: "Don't stress." If you're busy with the RealLife(tm) then
please don't worry about this little project. It won't run away.

I'll delete the test file and see if that's enough.

**Comment by torbengb** [comment]
Travis is happy now! Any reason to delay the merge @nocalla or can I go ahead?

**Comment by nocalla** [comment]
Fire at will!

**Comment by torbengb** [comment]
*BANG!*

**Pull request merged**